prov/net/src/xnet_progress.c

```diff
@@ -865,12 +865,22 @@ void xnet_progress_rx(struct xnet_ep *ep)
 			ret = ep->cur_rx.handler(ep);
 		}
 
-		if (OFI_SOCK_TRY_SND_RCV_AGAIN(-ret))
-			return;
+		if (OFI_SOCK_TRY_SND_RCV_AGAIN(-ret) ||
+		    ret == -OFI_EINPROGRESS_URING)
```

Comment:
Oh, hey, you added that check in this patch.  Can you fixup (or remove) the braces following the if statement.